### ISSUE TITLE
Improve new message push notifications

### ISSUE THREAD

**Issue by brianlovin** [withspectrum/spectrum#3182]
<img width="363" alt="screenshot 2018-05-29 14 04 07" src="https://user-images.githubusercontent.com/1923260/40685471-332df7d4-6349-11e8-9b2c-5a0de0fabf3a.png">

The push notification preview shows this for every subsequent reply in that thread - this is because of how we are buffering notifications, but it's very annoying to get this exact same push notification when different people are replying in my threads.

**Comment by mxstbr** [comment]
It's weird to get them in the reverse order with the newest at the top. I had it that way for a while but it felt very unexpected and unnatural to me?

**Comment by brianlovin** [comment]
I think that's the default behavior of the notifications pane. And regardless of order, the problem is that a user will see the same push notification 100x in an active conversation with the exact same preview title and preview text.

**Comment by brianlovin** [comment]
> (it's not the same preview title as the number of replies change either way) 

The number of replies at the end of the string is wrong though with our bundling. It will say 11 new replies even if I opened the app a minute ago when there were only 10 new replies. E.g. the bundler isn't considering that I saw the app between the 10th and 11th new messages.

> That's what I'm saying though, if we reverse the order of the messages in the notification itself it won't be the same preview text.

Then we should reverse it for sure - seeing the same preview text shouldn't happen for new notifications.
